src/services/gnosis/gnosisProtocol.service.ts

```diff
@@ -131,20 +131,19 @@ export default class GnosisProtocolService {
     try {
       const { amount, sellToken, buyToken, kind } = params;
 
-      const { baseToken, quoteToken } = getCanonicalMarket({
-        sellToken,
-        buyToken,
-        kind,
-      });
-      const market = `${toErc20Address(baseToken)}-${toErc20Address(
-        quoteToken
-      )}`;
-
-      const response = await axios.get<PriceInformation>(
-        `${this.baseURL}/markets/${market}/${kind}/${amount}`
+      const response = await axios.post<CowSwapQuoteResponse>(
+        `${this.baseURL}/quote`,
+        {
+          sellToken,
+          buyToken,
+          from: sellToken,
```

Comment:
I'm not sure what this parameter is for. I can use user account but we also want quotes when wallet is not connected.